kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiServer.scala

```diff
@@ -133,6 +134,13 @@ class KyuubiServer(name: String) extends Serverable(name) {
     if (conf.get(MetricsConf.METRICS_ENABLED)) {
       addService(new MetricsSystem)
     }
+
+    // initialize meta source
+    if (conf.get(FRONTEND_PROTOCOLS).map(FrontendProtocols.withName)
+      .contains(FrontendProtocols.REST)) {
```

Comment:
Is it the only scenario that we should initialize the meta store?

Reply:
This is need changing by kyuubi official